src/styles/index.scss

```diff
@@ -1,3 +1,147 @@
-body {
+// #region page
+@import "./variables";
+@import "https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap";
+
+* {
```

Comment:
use only `box-sizing: border-box` for `*` selector and reset default margin/paddings for those tag selectors you need